geoh5py/ui_json/input_file.py

```diff
@@ -519,10 +521,15 @@ def promote(self, var: dict[str, Any]) -> dict[str, Any]:
         """Convert uuids to entities from the workspace."""
         if self._geoh5 is None:
             return var
-
         for key, value in var.items():
             if isinstance(value, dict):
-                var[key] = self.promote(value)
+                if "groupValue" in value and "value" in value:
+                    var[key] = {
+                        "groupValue": self._uid_promotion(key, value["groupValue"]),
```

Comment:
Let's convert to snake case if possible. Otherwise our pydantic variable will have to keep it that way.